pkg/apis/v1alpha1/common.go

```diff
@@ -60,9 +91,19 @@ type ResourceReference struct {
 }
 
 type Source struct {
+	// Source code location in a git repository.
+	// +optional
 	Git *GitSource `json:"git,omitempty"`
+
 	// Image is an OCI image is a registry that contains source code
-	Image   *string `json:"image,omitempty"`
+	// OCI Image in a repository, containing the source code to
```

Comment:
Should this old doc be removed?

Reply:
good catch

Reply:
I commited a patch for this, not sure why this PR comment is replaced with <outdated> ?